README.md

```diff
@@ -104,9 +104,19 @@ seo()->twitterTitle('About us')
 By default, no favicon links will be included. You can manually enable the extension by calling:
 
 ```php
-seo()->favicon('path/to/logo.png');
+seo()->favicon();
```

Comment:
One thing I'm thinking about is that this could accept parameters in case the favicons are in different locations. Using the standard files is best, but there can be other cases and I guess if we're already implementing this then we can do it fully.

Check https://github.com/LeanAdmin/lean/blob/master/resources/views/components/layout/meta.blade.php#L5-L11 and https://github.com/LeanAdmin/lean/blob/master/assets/lean.php#L97-L117